she_codes_news/news/models.py

```diff
@@ -1,8 +1,18 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+User = get_user_model()
 
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+    # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="stories"
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    img_url = models.CharField(max_length=200)
```

Comment:
probably should have used a URLField rather than a CharField, comes with extra validation.
Also probably could have used null=True, blank=True to prevent forcing you to put in the value 0 for your migration.